app/assets/javascripts/home.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```

Comment:
I mean, you don't have to commit them after they're generated, and you can delete them afterwards. #46 is a ticket to fix the generator, but that's fairly low priority right now.